crates/node/src/interactive_cli/context.rs

```diff
@@ -508,6 +512,7 @@ impl ContextCommand {
                     println!("{} Default context set to: {}", ind, context_id);
                 }
             }
+            Commands::Identity(identity_cmd) => identity_cmd.run(node)?,
```

Comment:
<img src="https://github.com/user-attachments/assets/18e15538-e617-4424-9fc0-4ffa1ad8fd45" width="50%" />

```suggestion
            Commands::Identity(identity) => identity.run(node)?,
```